frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -62,7 +62,7 @@
 
 <div id="header"></div>
 <p class="pageHeader">
-    <span class="pageHeaderBigSpan">Components</span> <span class="pageHeaderSmallSpan">(${componentList.size()})</span>
+    <span class="pageHeaderBigSpan">Components</span> <span class="pageHeaderSmallSpan" id="componentCounter">(${componentList.size()})</span>
```

Comment:
I have added a new span and taken out the code concerning the parentheses, thanks.